validator/main.py

```diff
@@ -289,14 +295,13 @@ def validate(
         invalid_topics = set(metadata.get('invalid_topics', self._invalid_topics))
         all_topics = list(valid_topics | invalid_topics)
 
-        # throw if valid and invalid topics are empty
-        if not valid_topics:
+        # If there aren't any valid or invalid topics...
+        if len(all_topics) == 0:
             raise ValueError(
-                "`valid_topics` must be set and contain at least one topic."
+                "There must be at least one valid or invalid topic."
             )
```

Comment:
add a test that demonstrates using this validator while only supplying off topic topics.

Reply:
Fixed.  Added tests for only off-topic, on-topic, and with metadata overrides.